dapp/pages/_app.js

```diff
@@ -189,6 +197,19 @@ function App({ Component, pageProps, err }) {
             pauseOnFocusLoss
             pauseOnHover
           />
+          <script
```

Comment:
I think this sort of script tag would fit better in the _document file

Reply:
moved, thanks

Reply:
Late question here. What is the benefit/purpose of having this script in the _document file? We have the script in the _app file for every other deployment, and ideally, we keep this consistent across our stack.

Reply:
We have an unwritten convention where we put any javascript links or vanilla javascript tags in the `_document` as you can see [here for example](https://github.com/OriginProtocol/origin-dollar/blob/master/dapp/pages/_document.js#L63-L100) and _app.js contains react javascript components and modules imported via package manager. 

I don't see any technical benefits to this approach. And things would work equally well if they were in the `_app.js` file

Reply:
Sounds good. Thanks for the explanation!